src/index.ts

```diff
@@ -14,6 +14,601 @@
  * limitations under the License.
  */
 
-export async function date() {
-  console.warn('👻');
+const FULL_ISO_REG = /\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d{4,9}Z/;
```

Comment:
Is this available in a module anywhere? Could it fit in that regex GH org we found last week?

Reply:
Initially I had tried to use that, but that specific RegExp is only good for simple ISO strings, which is what the Date object returns.